infra/terraform/secure_pipelines/mob_sts_mock_pipeline.tf

```
@@ -0,0 +1,57 @@
+resource "aws_cloudformation_stack" "mob_sts_mock_pipeline" {
+    name = "mob-sts-mock-pipeline"
+
+    template_url = format(local.preformat_template_url,
+    "sam-deploy-pipeline",             # https://github.com/govuk-one-login/devplatform-deploy/tree/main/sam-deploy-pipeline
+    "VsTCw4M76yOZXOvXrX6_DrRBGcSFwqRk" # v2.66.0
+  )
+
+  // Default parameters. Can be overwritten by using the locals below.
+  parameters = merge(
+    {
+      Environment  = var.environment
+      SAMStackName = "sts-mock"
```

As with the existing stack this stack name should match the stack names of the `-tir` and `-pl` stacks. 

Either: 
This should be 
```suggestion
      SAMStackName = "mob-sts-mock"
```
OR
We rename the supporting stacks to `sts-mock-pl` and `sts-mock-tir`

---

Done.